source/github/repo.go

```diff
@@ -117,9 +117,10 @@ func (c *client) Enable(u *library.User, org, name, secret string) (string, erro
 	// create the hook object to make the API call
 	hook := &github.Hook{
 		Events: []string{
-			constants.EventPush,
-			constants.EventPull,
-			constants.EventDeploy,
+			eventPush,
```

Comment:
Similar to above, we should use the variables declared in `go-vela/types/constants` 👍 

https://github.com/go-vela/types/blob/80de769c682b6b87d09b6a76935cacea8ca8d78f/constants/event.go#L8-L23

Reply:
The goal was to separate out the Vela specific syntax from the source provider. While `push` and `pull_request` match, the comment event does not match the approved proposal and other event types (labels, reactions, etc) we may choose to have a different syntax in Vela than what he source provider provides. 

Also, should help us when we look to add other providers like GitLab. When I was looking through their [documentation on events](https://docs.gitlab.com/ee/api/events.html) it appeared they also their events will be slightly different from GitHub when we do constants. So, we thought of moving towards a design where the provider is responsible for its provider's specific syntax would be valuable.

Reply:
see my other comments above

Reply:
👍 I see the difference now!